CONTRIBUTING.rst

```diff
@@ -64,11 +64,10 @@ Ready to contribute? Here's how to set up `git_wrapper` for local development.
 
     $ git clone git@github.com:your_name_here/git_wrapper.git
 
-3. Install your local copy into a virtualenv. Assuming you have virtualenvwrapper installed, this is how you set up your fork for local development::
+3. Use pipenv to setup a new virtualenv. Pipenv will create a new virtualenv with packages at the correct versions. Assuming you have pipenv installed, this is how you set up your fork for local development::
 
-    $ mkvirtualenv git_wrapper
-    $ cd git_wrapper/
-    $ python setup.py develop
+    $ pipenv install --dev --ignore-pipfile
```

Comment:
Two questions here:
1. why ignore the pipfile?
2. why not use 'make dev' like we do in koji_wrapper?